doc/news/changes/minor/20190810Workshop

```diff
@@ -0,0 +1,8 @@
+Improved: Make a large number of prose, grammar, and various other improvements
+to the library.
+<br>
+(Ashna Aggarwal, Mario Zepeda Aguilar, Manaswinee Bezbaruah, Katherine Cosburn,
```

Comment:
I thought the format needs to be ```(name, name, name, date)``? Do our scripts handle newlines in the list of names?

Reply:
They do: I generated the doxygen and it looks fine (everything is on one line).

Reply:
The script @tjhei mentioned is the one we use for the release checklist and the release papers. It's in a separate repository. We'll probably have to adjust a few things by hand, but I think we'll remember when we need to do that.

Reply:
I'll take a look.